python/cdp_missoula_backend/scraper.py

```diff
@@ -54,7 +58,29 @@ def get_events(
     # (Encoder1_AF_2022-03-02-02-03.mp4)
     # The video link is https://video.isilive.ca/missoula/ + value from previous step
 
+    options = webdriver.ChromeOptions()
+    options.add_argument("--ignore-certificate-errors")
+    options.add_argument("--incognito")
+    options.add_argument("--headless")
+    driver = webdriver.Chrome("./chromedriver", options=options)
+
+    msla_url = "https://pub-missoula.escribemeetings.com/?Year=2022"
```

Comment:
TODO: Remove `?Year=2022`, the query param isn't retained when the driver makes the request, which is why line 71 is necessary